client/src/components/Dataset/services.ts

```diff
@@ -0,0 +1,77 @@
+import { fetcher, paths } from "schema";
+import { safePath } from "utils/redirect";
+
+const _getDatasets = fetcher.path("/api/datasets").method("get").create();
+type getDatasetsApiOptions = paths["/api/datasets"]["get"]["parameters"]["query"];
```

Comment:
I think the general convention for naming types is `PascalCase`, but we should probably agree on one style and bake it into eslint rules if there are not some already defined.

Reply:
Thanks and agreed, I did realize that later